src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,32 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init() throws ServletException {
+        myResource = MyCoolResource.openResource();
```

Comment:
```suggestion
        super.init();
        myResource = MyCoolResource.openResource();
```
"Do not forget to call init() and destroy() methods from the superclass when we override them"

Reply:
Mentor said, that it's not necessarily